lib/src/main/scala/spinal/lib/Stream.scala

```diff
@@ -1085,101 +1090,277 @@ object StreamFifo{
   def apply[T <: Data](dataType: T, depth: Int) = new StreamFifo(dataType,depth)
 }
 
-class StreamFifo[T <: Data](val dataType: HardType[T], val depth: Int) extends Component {
+/**
+  * Improved StreamFifo over the original implementation
```

Comment:
not sure we need to refer to the original one that does not exist anymore after this merge?

Reply:
Maybe mention: Fully redesigned in release 1.8.2 allowing improved timing closure.

Reply:
yes, doing it.